### ISSUE TITLE
The published mesh2d examples do not work.

### ISSUE THREAD

**Issue by Daiki48** [bevyengine/bevy#6714]
It didn't work with the current examples code, so I modified it in my own way.

This is the code that is published now.
https://github.com/bevyengine/bevy/blob/main/examples/2d/mesh2d.rs

```rust
use bevy::{prelude::*, sprite::MaterialMesh2dBundle};

fn main() {
    App::new()
        .add_plugins(DefaultPlugins)
        .add_startup_system(setup)
        .run();
}

fn setup(
    mut commands: Commands,
    mut meshes: ResMut<Assets<Mesh>>,
    mut materials: ResMut<Assets<ColorMaterial>>,
) {
    commands.spawn(Camera2dBundle::default());
    commands.spawn(MaterialMesh2dBundle {
        mesh: meshes.add(Mesh::from(shape::Quad::default())).into(),
        transform: Transform::default().with_scale(Vec3::splat(128.)),          // An error occurs here.
        material: materials.add(ColorMaterial::from(Color::PURPLE)),
        ..default()
    });
}
```
The version of `bevy` has been upgraded, so there may have been some changes.
So I have made this code work.

```toml
[dependencies]
bevy = "0.9.0"
```

```rust
use bevy::{
    prelude::*,
    sprite::MaterialMesh2dBundle
};

fn setup(
    mut commands: Commands,
    mut meshes: ResMut<Assets<Mesh>>,
    mut materials: ResMut<Assets<ColorMaterial>>
) {
    commands.spawn(Camera2dBundle::default());
    commands.spawn(MaterialMesh2dBundle {
        mesh: meshes.add(Mesh::from(shape::Quad::default())).into(),
        transform: Transform { translation: (default()), rotation: (default()), scale: (Vec3::splat(128.)) },    // Modified
        material: materials.add(ColorMaterial::from(Color::PURPLE)),
        ..default()
    });
}

fn main() {
    App::new()
    .add_plugins(DefaultPlugins)
    .add_startup_system(setup)
    .run();
}
```

I have a short programming history. I got this code to work but I don't know if it is the right code. Please let me know if there is a more beautiful solution.
I do not know how to contribute to the OSS.
A beginner like me would be glad to have a full set of examples.I wanted to contribute to bevy, so I created the issues.
Thanks for the great project!!

**Comment by rparrett** [comment]
Could you please show the error you are seeing?

The code in your first code block runs without error for me both in bevy 0.9 and with bevy's main branch.

Examples in the `main` branch are always up-to-date with code changes in the `main` branch. This means that they may not work with the latest released version of bevy if breaking changes have been introduced.

You can see examples that work with the latest release by looking at the `latest` tag on github.

https://github.com/bevyengine/bevy/blob/latest/examples/2d/mesh2d.rs

**Comment by Daiki48** [comment]
@alice-i-cecile Thanks for showing us around. I have never PR'd before, so I will check the URL address you gave me. If I need to do a PR, I will try to implement it.

@rparrett Thanks for confirming the EXAMPLES. Also, thanks for taking a look at this issue. The error is this image.

<img width="960" alt="20221122_bevy_Transform" src="https://user-images.githubusercontent.com/102207294/203179044-cb2fdd2c-efc8-40bd-a5fe-f4127d436619.png">

Then I realized my error. I had made a mistake in writing the Sorry for my elementary mistake.

The `Transform::default()` part was written as `Transform.default()`.
I write all the code myself for the RUST study. This has resulted in a description error.

This issue is closed.
